### PR TITLE
288 set pose with new zeros

### PR DESCRIPTION
**Pull request by glannuzel**
Based on #292

### CODE REVIEW

File: src/reachy2_sdk/reachy_sdk.py [hunk 1]
```diff
@@ -615,19 +615,19 @@ def is_off(self) -> bool:
 
     def set_pose(
         self,
-        common_pose: str = "zero",
+        common_pose: str = "default",
```
Comment (FabienDanieau) [inline src/reachy2_sdk/reachy_sdk.py:618]: There are references to the zero poses in the examples, notebook and tests to be changed also

File: src/reachy2_sdk/reachy_sdk.py [hunk 2]
```diff
@@ -615,19 +615,19 @@ def is_off(self) -> bool:
...
         duration: float = 2,
         interpolation_mode: str = "minimum_jerk",
     ) -> GoToHomeId:
-        """Send all joints to 0 in specified duration.
+        """Send all joints to standard positions in specified duration.
 
-        common_pose can be 'zero' or 'elbow_90', default is set to 'zero'.
-        Setting wait_for_goto_end to False will cancel all gotos on all parts and immediately send the 0 commands.
-        Otherwise, the 0 commands will be sent to a part when all gotos of its queue has been played.
+        common_pose can be 'default', arms being straight, or 'elbow_90'.
```
Comment (FabienDanieau) [inline src/reachy2_sdk/reachy_sdk.py:625]: Why not straight_arms instead of default then? The name will be more explicit

**Review by FabienDanieau — CHANGES_REQUESTED** [review]
Tests ok. But some examples will fail

**Comment by glannuzel** [comment]
Modif merged with #272

**Pull request closed without merging**